src/main/java/seedu/address/ui/DeadlineTaskListCard.java

```diff
@@ -31,7 +32,7 @@ public DeadlineTaskListCard(DeadlineTask deadlineTask, int index) {
             indexLabel.setText(IndexPrefix.DEADLINE.getPrefixString() + index + ". ");
             nameLabel.setText(deadlineTask.getName().toString());
             dueLabel.setText(deadlineTask.getDue().toString());
-            if (deadlineTask.isFinished()) {
+            if (!(new DeadlineNotFinishedPredicate()).test(deadlineTask)) {
```

Comment:
The original is slightly simpler, compared to having to create a new `DeadlineNotFinishedPredicate` object just to use its `test()` method.